Source/Details/ASDataController.mm

```diff
@@ -437,6 +437,29 @@ - (void)waitUntilAllUpdatesAreCommitted
   [self _scheduleBlockOnMainSerialQueue:^{ }];
 }
 
+- (BOOL)isProcessingUpdates
+{
+  ASDisplayNodeAssertMainThread();
+  long waitResult = dispatch_group_wait(_editingTransactionGroup, DISPATCH_TIME_NOW);
+  return (waitResult == 0 ? NO : YES);
```

Comment:
@Adlai-Holler do you think we should perform a setNeedsLayout in the case that the queues are drained, before returning NO? This would technically be needed to ensure all information is flushed to / reflected in the UICollectionViewLayout...